src/components/tables/DataTable.vue

```diff
@@ -38,7 +38,7 @@ export interface Props {
   /**
    * model for selected rows, add a v-model to support row selection
    */
-  value?: string[];
+  value?: TableRow[TableRowKey][];
```

Comment:
@tewshi would that result to any since we don't have any generics in 2.7?

Reply:
correct...